.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,4 @@
+<!-- Keep the progress checklist.  -->
 ## Progress
```

Comment:
What makes me lean towards removing it is that for issues like this [one](https://github.com/cerner/terra-core/issues/403), most of the checkboxes are not applicable or can be identified by other ways. I feel like setting up columns in our project board or using labels we can communicate the progress status.